<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleMgrDlg</class>
 <widget class="QDialog" name="ModuleMgrDlg">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>模块管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="UI.qrc">
    <normaloff>:/res/IcoModule.ico</normaloff>:/res/IcoModule.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,12">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="1" column="0">
      <widget class="QPushButton" name="NewModuleButton">
       <property name="text">
        <string>新增模块</string>
       </property>
       <property name="icon">
        <iconset resource="UI.qrc">
         <normaloff>:/res/ImgPlus.png</normaloff>:/res/ImgPlus.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="DelModuleButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>删除模块</string>
       </property>
       <property name="icon">
        <iconset resource="UI.qrc">
         <normaloff>:/res/ImgDel.png</normaloff>:/res/ImgDel.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QListWidget" name="ModuleListWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="ModuleInfoTabWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>常规</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableWidget" name="InfoTableWidget">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>downloadUrl</string>
          </property>
          <property name="toolTip">
           <string>远程文件根目录</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>uploadUrl</string>
          </property>
          <property name="toolTip">
           <string>上传根路径</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>remoteVersionUrl</string>
          </property>
          <property name="toolTip">
           <string>远程版本文件</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>remoteManifestUrl</string>
          </property>
          <property name="toolTip">
           <string>远程清单文件</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>值</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>目录</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="2">
        <widget class="QPushButton" name="DelDirButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>删除目录</string>
         </property>
         <property name="icon">
          <iconset resource="UI.qrc">
           <normaloff>:/res/ImgDel.png</normaloff>:/res/ImgDel.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="NewDirButton">
         <property name="text">
          <string>新增目录</string>
         </property>
         <property name="icon">
          <iconset resource="UI.qrc">
           <normaloff>:/res/ImgPlus.png</normaloff>:/res/ImgPlus.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QListWidget" name="DirsListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>文件</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="2">
        <widget class="QPushButton" name="DelFileButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>删除文件</string>
         </property>
         <property name="icon">
          <iconset resource="UI.qrc">
           <normaloff>:/res/ImgDel.png</normaloff>:/res/ImgDel.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="NewFileButton">
         <property name="text">
          <string>新增文件</string>
         </property>
         <property name="icon">
          <iconset resource="UI.qrc">
           <normaloff>:/res/ImgPlus.png</normaloff>:/res/ImgPlus.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QListWidget" name="FilesListWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="UI.qrc"/>
 </resources>
 <connections/>
</ui>
